### DISCUSSION TITLE
✅ chaoss-ospo-wg-discuss Slack Channel

### DISCUSSION THREAD

**Discussion by hyandell** [todogroup/ospology#218] [Community Proposals]
Some of us at TODOGroup joined the CHAOSS OSPO Working Groups call to discuss OSPO metrics. Continuing that discussion would be great. Can we have a Slack channel to continue that discussion please?

**Comment by germonprez** [comment]
+1 
Would love to see this channel

**Reply by will-ww** [reply]
+1

**Comment by anajsana** [comment]
Thanks for submitting this channel creation proposal @hyandell! @germonprez I think doing this will solve your question about what could be the best channel to use for TODO / CHAOSS communication 👍  A few comments / suggestions on my end: 

* I would suggest calling it `#wg-ospometrics-chaoss`. This way, in case other Working Groups, are created with other open source communities (e.g ISC, FINOS, LF Energy) in the future, can be initiated using the `#wg-WorkingGroupTitle-OpenSourceCommunity` format, so `wg` channels will be easier to find. 

* In the same way we mirror slack conversations on GH Discussions for those who can't access slack, I would suggest recording the suggestions and ideas from that channel anonymously into: https://github.com/todogroup/ospology/discussions/categories/ospo-wishlist

* FYI, CHAOSS APAC and OSPOlogy APAC Sync meetings had a parallel conversation related to this. @xiaoya-yaya and I are working on a proposal to submit here to align both calls in 2023 and work on **OSPO metrics for OSPO sustainability and continuity in ASIA-PACIFIC orgs** and record the meeting notes to report to the CHAOSS WG repo [here](https://github.com/todogroup/ospology/tree/main/meetings/Sync Chapters).

* Last but not least, please wait until one of our TODO SC review this. (cc @ashleywolf who AFAIK was also involved in this discussion). Once reviewed, I will proceed and create the slack channel!

Maybe we can also find a way to have calendar notifications in that channel for that specific CHAOSS call? Do you know if it's possible to do that without notifying the whole CHAOSS calendar?

**Comment by germonprez** [comment]
Thanks @anajsana 

- No problem on the naming approach from me. 
- I didn't know that you could mirror Slack Conversations to GH. That's cool. 
- Good to know on the work with @xiaoya-yaya 

Regarding the calendar, could you just make a new event on the TODO calendar?

**Reply by anajsana** [reply]
Creating a new event in TODO calendar with the CHAOSS meeting link is a great idea. Will do! Thanks for the feedback! 😊

**Reply by germonprez** [reply]
Just a note on this @anajsana -- we are taking a break during December in the CHAOSS project. The next CHAOSS-TODO group meeting is Jan 12, 2023

**Comment by souravcipher** [comment]
This sounds exciting! 

Looking forward to it.

**Comment by ashleywolf** [comment]
Really excited to see this! I joined the first call and would be great to have more representation from TODO members. This CHAOSS working group is focused on OSPO value metrics and if you have or want to build a data-driven OSPO (or open source success metrics within OSPO) this will be a great group to participate in.

**Comment by FelixFaassen-TomTom** [comment]
I would also be very interested in this. Looking forward to it.

**Comment by anajsana** [comment]
The new slack channel has been created! people can search for `#wg-ospometrics-chaoss` within TODO slack and join the discussion there 👍
